src/lib/forward.ts

```diff
@@ -119,14 +119,14 @@ export class EventForwarder {
       this.clients.clients.forEach((client) => {
         // Copy arguments for each client so transforms don't impact all clients
         const newArgs = { ...args };
-        if ("prevValue" in newArgs) {
+        if ("prevValue" in newArgs && "metadata" in newArgs.prevValue) {
           newArgs.prevValue = { ...newArgs.prevValue };
```

Comment:
Instead of copying, we should probably also specify each value here. Maybe a `dumpValue` function?

Reply:
I actually think I misunderstood what's passed in to `prevValue` and `newValue` entirely. I thought these were objects but I think they are just string, number, etc. that represents the old value and the new value for that particular `Value`. I will update the PR to reflect this